dali/python/nvidia/dali/pipeline.py

```diff
@@ -80,6 +84,7 @@ def __init__(self, batch_size = -1, num_threads = -1, device_id = -1, seed = -1,
                  exec_pipelined=True, prefetch_queue_depth=2,
                  exec_async=True, bytes_per_sample=0,
                  set_affinity=False, max_streams=-1, default_cuda_stream_priority = 0):
+        self._sinks = []
```

Comment:
Please describe how are the sinks intended to be collected, because the logic may be a little bit hard to discover after some time.